### ISSUE TITLE
[New Rule] Avoid 4 or more repeated empty print()

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#136]
## Explanation

3 elements are ok to recognize, but when you quickly have a look it's hard to distinguish 4 repeated lines from 5 repeated lines. Also, it clutters the code

## Example

```python
# Bad
print()
print()
print()
print()

# Good
print("\n" * 4)
```
